### ISSUE TITLE
Increase Haggle AP Cost

### ISSUE THREAD

**Issue by ghost** [Arx-Game/arxcode#235]
**Is your feature request related to a problem? Please describe.**
The current cost of haggling, at 5 AP, is currently a much easier way for people to generate resources, silver or luxury goods than by the work command and other means.  

Work costs 15 AP and typically generates far fewer resources except for top tier players.

**Describe the solution you'd like**
Consider increasing the base haggle AP cost from 5.
Reasonable proposals:
* to 15, in line with work commands
* to 25+ in line with investigation commands. Possible to be a base 25 or (50 - Haggle ranks, min 25)

**Additional context**
Due to the haggle system in general, it's easy for someone to convert 150 AP a week into 30 haggle rolls. With a very easy outlay of XP for a committed haggler, these characters can turn over individually what an entire noble house or organization makes in a week with its full membership using work commands.

Currently the 5 AP is just too low compared to other AP uses.

**Comment by dreamsofgold** [comment]
Haggle is not condensed to a single process; it has multiple rolls, and the difficulties can scale with the material you're trying to purchase. 

Hagglers, and merchants in general, take more than a few rolls to make even with their investments. I don't see how there would be any benefit to balancing by making haggling so unattractive an option that it costs the same as an investigation or a work roll. 

It would likely be a good idea to escalate difficulties based on how many haggles were done in a day, up to a maximum, per type. 

Edit: Before I forget, +work is also heavily affected by the org's modifiers for each type of resource, and they can, pretty much, make your AP investment break even. A lucky week for a character spending 200 AP can make around 600K worth of resources at a very basic social clout level.

**Comment by Briarsgrove** [comment]
Agreed. As a haggler myself I can spend sometime 30AP + to get rare materials or better prices. The initial roll is 5AP to FIND the items. Then if you get a low roll on your haggle for price either selling or buying you spend another 5 AP to better it. So that is 15AP right there. I have even done an additional roll and gotten the same as before so had to pull a few more. So sometimes a single transaction will cost my player 25-35AP.
